.gitignore

```
@@ -100,3 +100,4 @@ src/x_build_libs.ocp
 # dev setup
 /.ocamlinit
 /release/out
+.github/actions
```

Without the `/` this can match any prefix:
```suggestion
/.github/actions
```